### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by glorymugizi**
Added Html file
Styled the heading
Added linters
Updated the readme file

Thank you for reviewing my pull request

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @glorymugizi   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to add a descriptive PR description. It makes your repo looks more professional. :wink: 

   - Example of good PR
image.

- [ ] Kindly make sure to document your README professionally. Please use the [Microverse README template](https://github.com/microverseinc/readme-template) and customize it. :wink:

- [ ] Kindly make sure the `.gitignore` file is presented in your repo, where we must have all ignore files which are `node_modules` and `test.md` as required. :wink:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by glorymugizi** [comment]
> ## Hello @glorymugizi , 👋
> ### I'm Levy, your PR reviewer
> Good job so far 👍 There are some issues that you still need to work on to go to the next project but you are almost there!
> 
>  [ 68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966 ](https://camo.githubusercontent.com/77566116408d88259ddadf7605a02b67a4e825e002a39682ca5f2e8bca8c4d64/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) [ ](https://camo.githubusercontent.com/77566116408d88259ddadf7605a02b67a4e825e002a39682ca5f2e8bca8c4d64/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)
> 
> ## Status: REQUIRED CHANGES ♻️
> _Check the comments under the review._
> 
> * [ ]  Kindly make sure to add a descriptive PR description. It makes your repo looks more professional. 😉
>   
>   * Example of good PR
>     image.
> * [ ]  Kindly make sure to document your README professionally. Please use the [Microverse README template](https://github.com/microverseinc/readme-template) and customize it. 😉
> * [ ]  Kindly make sure the `.gitignore` file is presented in your repo, where we must have all ignore files which are `node_modules` and `test.md` as required. 😉
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @levy002 in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Hello Microverse!</h1>
+  </body>
+</html>
```
Comment (glorymugizi) [inline index.html:13]: Hi @levy002 I have tried to go through the requested changes and everything looks fine only the gitignore, test.md and readme.md files were accidentally sent to the main branch. Perhaps you can check them out and advise accordingly and guide me on the way forward. I seem stuck.

**Comment by glorymugizi** [comment]
Hi @levy002 I have tried to go through the requested changes and everything looks fine only the gitignore, test.md and readme.md files were accidentally sent to the main branch. Perhaps you can check them out and advise accordingly and guide me on the way forward. I seem stuck. 

Thank you

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @glorymugizi😃  , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Commit messages meaningful ✔️ 
Descriptive pull request title and summary ✔️

## Required Changes ♻️

- [ ] - [ ] @levy002, I'm so sorry but I couldn't find .gitignore file in the main branch. Kindly add .gitignore file to your `development ` branch and add test.md and node_modules. Your test.md file should have the following content.

> node_modules/
    test.md

Please take note for subsequent review, you should have used a more descriptive name for your feature branch, shouldn't have been `development`. ℹ️ 

- [ ] Please add your repository link to the setup section of your Readme file; also don't forget to add details for the installation and usage section. Thanks. 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by glorymugizi** [comment]
Hi @OyePriscilla I have updated my README but the checks are showing zero instead of 4.
Please kindly check it out. Is it ok for me to send a review as this is my last review?

Thank you.

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @glorymugizi        👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉 

## Highlights
✔️ Professional Pull Request 👍
✔️ Professional Readme 👍  
## Optional suggestions
- [ ] [OPTIONAL] **There's no linter checking for this pull request 😨** 
 It would be better to correctly set up [ linter for HTML and CSS](https://github.com/microverseinc/linters-config) and the Linter result should be green. AKA: Linter should report NO errors.
image


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**